docs/knowledge_base/demos/sales_dashboard.md

```diff
@@ -0,0 +1,52 @@
+Welcome to Taipy's Sales Dashboard demo, where we highlight the impressive capabilities of our 
+platform in data visualization. In this demonstration, we will explain the description and 
+operation of our Sales Dashboard, demonstrating how you can easily use Taipy to design 
+interactive dashboards for your Excel data.
+
+# Understanding the Sales Dashboard
+The Sales Dashboard is a versatile tool that streamlines the task of visualizing data from Excel 
+files. This demo harnesses Taipy's technology, allowing users to extract data from Excel and 
+present meaningful insights. Users can apply filters based on city, customer, and gender, making 
+it straightforward to analyze metrics tailored to various groups and locations.
+
+# The Sales Dashboard Demo
+## Filtering Data for Insights
+Taipy's Sales Dashboard allows users to filter Excel data for valuable insights. Whether you 
+want to examine sales metrics for customers in a particular city or explore sales metrics based 
+on gender, this tool empowers you to understand your data better.
+
+## Dynamic Charts for Trend Analysis
+The Sales Dashboard includes two interactive charts: one for sales by hour and another for sales 
+by product. These charts provide visual representations of your data, making it easy for users 
+to spot trends, patterns, and connections. Whether you're examining sales trends over time or 
+evaluating product performance, these charts deliver valuable insights.
+
+# Powered by Taipy
+The Sales Dashboard demo showcases Taipy's proficiency in streamlining intricate tasks and 
+converting data into actionable information. With its user-friendly filtering choices and 
+dynamic charting features, Taipy enables users to swiftly and effortlessly produce data-driven 
+dashboards.
+
+# How to Use Taipy for Sales Dashboard Creation
+To create interactive and insightful sales dashboards with Taipy's Sales Dashboard demo, follow 
+these simple steps:
+
+1. Visit Taipy [Sales Dashboard Demo](https://sales-dashboard.taipy.cloud/).
+
+2. Upload your Excel file containing the sales data you wish to visualize.
```

Comment:
Where? What kind of data and format is expected in the Excel file?

Reply:
Indeed. It seems like something has changed. 
In the introduction on the [website](https://www.taipy.io/project/sales-dashboard/) it's written: Demo Sales Dashboard demonstrates how Taipy can read an Excel file and show interesting results. This will be the base of our DataViz application.